src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitSchedulerService.kt

```diff
@@ -119,11 +121,16 @@ class VisitSchedulerService(
       }
   }
 
-  // convenience method to retrieve all session templates to support development
   fun getSessionTemplates(): List<SessionTemplateDto> {
     return sessionTemplateRepository.findAll().sortedBy { it.startDate }.map { SessionTemplateDto(it) }
   }
 
+  @Throws(TemplateNotFoundException::class)
```

Comment:
Out of interest, are the throws annotations just mainly for documentation? Or does intelliJ use them in someway?